runtimes/peregrine/src/lib.rs

```diff
@@ -1017,11 +1017,30 @@ impl OnRuntimeUpgrade for SchedulerMigrationV3 {
 			"Scheduler migrated to version {:?}",
 			Scheduler::current_storage_version()
 		);
-
 		Ok(())
 	}
 }
 
+// Migration for ParachainStaking pallet to mutate CandidatePool to be
+// a CountedStorageMap and switch to new Pallet StorageVersion paradigm.
+pub struct ParachainStakingMigrationV7;
```

Comment:
Could we not move this struct to the stacking pallet and just import it from there?